analysis_runner/README.md

```diff
@@ -2,9 +2,13 @@
 
 ## CLI
 
-There are two ways to run the analysis-runner CLI: with the `--repository` parameter, and without:
+There are two ways to run the analysis-runner CLI:
 
-1. Omitting the `--repository` parameter: use the repository of the local directory that you're in: (a) get the repository name from the git remote; (b) use the commit of HEAD (if the `--commit` parameter is omitted); (c) Make the script path relative to the root of the git repository.
+- **Omitting** the `--repository` parameter: the repository containing the local
+  directory that you're in is used. The repository name will be retrieved from
+  the git remote URL, while the commit of HEAD is used (if the `--commit`
+  parameter is omitted). The script path is made relative to the root of the git
```

Comment:
```suggestion
- **Omitting** the `--repository` parameter: uses the repository of the current directory. The repository name will be retrieved from
  the git remote URL, while the commit of HEAD is used (if the `--commit`
  parameter is omitted). The script path is made relative to the root of the git
```

Small suggestion, because I phrased it pretty terribly initially.